programs/marinade-finance/src/state/initialize.rs

```diff
@@ -85,13 +85,6 @@ impl<'info> Initialize<'info> {
         self.check_reserve_pda()?;
         self.check_msol_mint()?;
         self.check_treasury_accounts()?;
-        check_owner_program(
```

Comment:
This check can not be removed yet (because it is not checked by anchor). Will release another vision of operational_sol_account concept after some research because for now it looks like it has some yellow level security problem